### PR TITLE
allows build target to display when using presets

### PR DESCRIPTION
**Pull request by snehara99**
The build target shows in the Project Status View in the side panel now and mimics the same behaviour of the option in the status bar. Below are before and after screenshots when using CMake Presets.

Before (can't select build target):
image

After (can select build target):
image

### CODE REVIEW

**Comment by gcampbell-msft** [comment]
@snehara99 Before approving, could you help me understand exactly what the change is by putting some screenshots in of before and after? 

The changes seem fine, but I want to make sure I understand it well before I approve. Thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@
 Features:
 - Added support for the CMake Debugger. [#3093](https://github.com/microsoft/vscode-cmake-tools/issues/3093)
 
+## 1.14.35
```
Comment (benmcmorran) [inline CHANGELOG.md:7]: I think you can merge this with the 1.15 section. I don't think we're planning to do another patch release before 1.15.

**Review by benmcmorran — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (snehara99) [reply CHANGELOG.md:7]: sounds good

**Review by gcampbell-msft — APPROVED** [review]
